UI/window-basic-main-outputs.cpp

```diff
@@ -1573,8 +1570,6 @@ AdvancedOutput::AdvancedOutput(OBSBasic *main_) : BasicOutputHandler(main_)
 	obs_encoder_release(streamAudioEnc);
 
 	id = "";
-	int vodTrack =
-		config_get_int(main->Config(), "AdvOut", "VodTrackIndex") - 1;
 	streamArchiveEnc = obs_audio_encoder_create(streamAudioEncoder,
 						    ADV_ARCHIVE_NAME, nullptr,
 						    streamTrack, nullptr);
```

Comment:
It's bug, will PR a fix.

`streamArchiveEnc` should use it.

Reply:
Fix by #8510

Reply:
#8510 has been merged. Once this change is reverted, this PR can also be merged.

Reply:
Reverted that particular change.